cluster/kubernetes/doc.go

```diff
@@ -3,5 +3,4 @@ Package kubernetes provides implementations of `Cluster` and
 `Manifests` that interact with the Kubernetes API (using kubectl or
 the k8s API client).
 */
-
```

Comment:
Was this intentional?

Reply:
Yes, docs ain't docs unless they're immediately before the package declaration (although arguably it doesn't belong in this PR -- it's incidental).